spec/rusty_json_schema_spec.rb

```diff
@@ -63,9 +63,9 @@
 
       it "returns validation errors" do
         expect(validator.validate(event)).to eq [
-          "'\"rusty\"' is not of type 'number'",
-          "'1' is not of type 'string'",
-          "'baz' is a required property"
+          %("rusty" is not of type "number" at /bar),
```

Comment:
Just a matter of taste but I would format this as follows:

`property "foo": "rusty" is not of type "number"`

Reply:
makes sense! I'll set it to `path "/foo": ...` and `at root: ...`, does it make sense? @mobileka

Reply:
or even better `path "/": ...`

Reply:
Yup, I would also vote for `"/"` 👍